server/workflows/provision_device.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "provision_device",
+    "description": "Provision device.",
+    "version": 1,
+    "tasks": [
+        {
+            "name": "create_device_inventory",
+            "taskReferenceName": "create_device_inventory",
+            "inputParameters": {
+                "http_request": {
+                    "uri": "http://mender-inventory:8080/api/internal/v1/inventory/devices",
+                    "method": "POST",
+                    "contentType": "application/json",
+                    "body": "${workflow.input.device}",
+                    "headers": {
+                        "X-MEN-RequestID": "${workflow.input.request_id}",
+                        "Authorization": "${workflow.input.authorization}"
+                    }
+                }
+            },
+            "type": "HTTP"
+        },
+        {
+            "name": "device_accepted",
+            "taskReferenceName": "device_accepted",
+            "inputParameters": {
+                "sink": "conductor",
+                "ref_flow": "${workflow.input.ref_flow}",
+                "request_id": "${workflow.input.request_id}",
+                "authorization": "${workflow.input.authorization}",
+                "device": "${workflow.input.device}"
```

Comment:
any chance it's possible to slip in tenant id if available ?

Reply:
tenant id is in the authorization header
will this be enough?

Reply:
we can extract it, or even fetch it with a call to tenantadm so there are options,
if it's effort to add it's nice to have it attached

Reply:
it's hard to implement this not knowing exact requirements
maybe we should go back to requirements?
it looks like we need tenant id here, but I had no idea that we need it when I've been starting (end even finishing) this task;
if we need tenant id in this workflow event (and workflow), maybe there is something more we need there?
and maybe we don't need authorization header in the event?

Reply:
device id is inside device object; maybe we should extract it too?

Reply:
@maciejmrowiec @GregorioDiStefano can you guys just spec out the event? would:

- request_id
- tenant_id
- device_id

work?

Reply:
talked it over with Greg - we'll close it for now, and perhaps spin off an 'improvement' task later.